docs/multicluster/architecture.md

```diff
@@ -106,3 +108,52 @@ Multi-cluster admins can also specify certain ClusterNetworkPolicies to be repli
 ClusterSet. The ACNP to be replicated should be created as a ResourceExport in the leader cluster, and
 the resource export/import pipeline will ensure member clusters receive this ACNP spec to be replicated.
 Each member cluster's Multi-cluster Controller will then create an ACNP in their respective clusters.
+
+## Antrea Multi-cluster Service Connectivity
+
+Antrea Multi-cluster supports Service connectivity directly since 1.7.0. It introduces a new feature gate
```

Comment:
I feel no need to mention feature gate or particular component configurations in the architecture doc.

Reply:
Removed